### PR TITLE
fix(docs): content review #5

### PR DESCRIPTION
**Pull request by jcirinosclwy**
### Description

Content review, fixing typos.

### CODE REVIEW

File: console/my-account/how-to/open-a-support-ticket.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ categories:
   - console
 ---
 
-Scaleway provides 24/7 basic technical support by ticket for free. You can contact the technical assistance directly from your console. You can [upgrade your support level](/console/my-account/how-to/configure-support-plans/) to profit from phone support and a dedicated account manager.
+Scaleway provides 24/7 basic technical support by ticket for free. You can contact our Support team directly from the console. You can [upgrade your support level](/console/my-account/how-to/configure-support-plans/) to profit from phone support and a dedicated account manager.
```
Comment (RoRoJ) [inline console/my-account/how-to/open-a-support-ticket.mdx:16]: Maybe we should also change "profit from"

File: faq/elastic-metal.mdx [hunk 2]
```diff
@@ -38,7 +38,7 @@ We provide several ranges of Elastic Metal servers, each of them designed to mee
...
+Elastic Metal servers are available on two different billing plans: hourly or monthly. In both cases, you will receive a monthly invoice calculated at the end of each month and based on your resource consumption during that period (number of hours allocated to your account, if you selected hourly billing, or monthly price, if you chose a monthly billing). However, to benefit from the more advantageous monthly pricing, you will pay a one-time commitment fee equivalent to one month of subscription. To reward your long-term commitment, if you rent your Elastic Metal server for more than 12 months, we will give you Scaleway cloud credits equivalent to your commitment fee.
```
Comment (RoRoJ) [inline faq/elastic-metal.mdx:41]:
```suggestion
Elastic Metal servers are available on two different billing plans: hourly or monthly. In both cases, you will receive a monthly invoice calculated at the end of each month and based on your resource consumption during that period (number of hours allocated to your account, if you selected hourly billing, or monthly price, if you chose monthly billing). However, to benefit from the more advantageous monthly pricing, you will pay a one-time commitment fee equivalent to one month of subscription. To reward your long-term commitment, if you rent your Elastic Metal server for more than 12 months, we will give you Scaleway cloud credits equivalent to your commitment fee.
```

File: tutorials/jitsi-docker/index.mdx [hunk 3]
```diff
@@ -47,12 +47,12 @@ This tutorial explains how to install the Jitsi Meet solution on an Instance usi
...
     git clone https://github.com/jitsi/docker-jitsi-meet && cd docker-jitsi-meet
     ```
-5. Copy the `env.example` file to create a environment (`.env`) configuration and create the required configuration directories:
+5. Copy the `env.example` file to create an environment (`.env`) configuration and create the required configuration directories:
     ```
     cp env.example .env
     mkdir -p ~/.jitsi-meet-cfg/{web/letsencrypt,transcripts,prosody,jicofo,jvb}
     ```
-6. Open the `.env` file in a text editor and edit the basic settings as following:
+6. Open the `.env` file in a text editor and edit the basic settings as the following:
```
Comment (RoRoJ) [inline tutorials/jitsi-docker/index.mdx:55]:
```suggestion
6. Open the `.env` file in a text editor and edit the basic settings as follows:
```

File: tutorials/jitsi-docker/index.mdx [hunk 4]
```diff
@@ -93,7 +93,7 @@ This tutorial explains how to install the Jitsi Meet solution on an Instance usi
     - 10000/udp for RTP media over UDP
 8. Run `docker-compose up -d` to download and launch the application.
     <Message type="note">
-      If you want to use the Etherpad option with Jisi Meet, configure the Etherpad section in the `.env` file and start the docker container as following: `docker-compose -f docker-compose.yml -f etherpad.yml up`.
+      If you want to use the Etherpad option with Jisi Meet, configure the Etherpad section in the `.env` file and start the docker container as the following: `docker-compose -f docker-compose.yml -f etherpad.yml up`.
```
Comment (RoRoJ) [inline tutorials/jitsi-docker/index.mdx:96]:
```suggestion
      If you want to use the Etherpad option with Jisi Meet, configure the Etherpad section in the `.env` file and start the docker container as follows: `docker-compose -f docker-compose.yml -f etherpad.yml up`.
```

File: tutorials/setup-k8s-cluster-rancher/index.mdx [hunk 5]
```diff
@@ -16,9 +16,9 @@ dates:
 
 ## Rancher Overview
 
-Rancher is an open-source container management platform, providing an graphical interface making container management easier.
+Rancher is an open-source container management platform, providing a graphical interface making container management easier.
 
-The GUI of Rancher makes it easy to manage secrets, handle roles and permissions. It allows to scale nodes and pods, set up load balancers without the requirement of a command line tool or the editing of hard to read YAML files.
+The GUI of Rancher makes it easy to manage secrets, handle roles and permissions. It allows scaling nodes and pods, set up load balancers without the requirement of a command line tool or the editing of hard to read YAML files.
```
Comment (RoRoJ) [inline tutorials/setup-k8s-cluster-rancher/index.mdx:21]: To me, a more natural-sounding correction for this structure (in other places too) would either be:
- It allows you to scale node and pods
OR
- It allows the scaling of nodes and pods

(also the other verbs later in the sentence then need to change form accordingly, either way)

File: tutorials/ruby-on-rails/index.mdx [hunk 6]
```diff
@@ -100,7 +100,7 @@ We will call this user `deploy`.
 
 ## Installing and configuring of PostgreSQL
 
-Ruby on rails application typically use a SQL database for data persistence.
+Ruby on Rails application typically use a SQL database for data persistence.
```
Comment (nerda-codes) [inline tutorials/ruby-on-rails/index.mdx:103]:
```suggestion
Ruby on Rails application typically uses an SQL database for data persistence.
```

File: tutorials/setup-k8s-cluster-rancher/index.mdx (hunk 5 above)
Comment (nerda-codes) [inline tutorials/setup-k8s-cluster-rancher/index.mdx:21]:
```suggestion
The GUI of Rancher makes it easy to manage secrets, handle roles and permissions. It allows to scale nodes and pods, set up load balancers without needing a command line tool or to edit hard to read YAML files.
```

File: tutorials/odoo-easy-deploy/index.mdx [hunk 7]
```diff
@@ -184,5 +184,5 @@ The Odoo image [documentation](https://hub.docker.com/_/odoo) tells us that by m
 You can now start installing apps, configuring your settings and using Odoo. Check out the [official Odoo documentation](https://www.odoo.com/documentation/16.0/) for more help. 
 
 <Message type="note">
-  Keep in mind that the easy deploy feature was designed to help you get started with Kubernetes: we do not recommend using this feature for production use for two reasons: First, because it requires your container image to be publicly available, i.e. without access restrictions. And second, because it does not allow you to do such things as add persistent volumes and link several containers, for example, without fine-tuning some settings yourself.
+  Keep in mind that the Easy Deploy feature was designed to help you get started with Kubernetes: we do not recommend using this feature for production use for two reasons: First, because it requires your container image to be publicly available, i.e. without access restrictions. And second, because it does not allow you to do such things as add persistent volumes and link several containers, for example, without fine-tuning some settings yourself.
```
[the remark this answers is not in the document]
Comment (jcirinosclwy) [reply tutorials/odoo-easy-deploy/index.mdx:187]: On the handbook it is 'Easy Deploy'. Did we establish different rules according to the usage?

File: tutorials/setup-k8s-cluster-rancher/index.mdx (hunk 5 above)
Comment (jcirinosclwy) [inline tutorials/setup-k8s-cluster-rancher/index.mdx:21]:
```suggestion
The GUI of Rancher makes it easy to manage secrets, handle roles and permissions. It allows you to scale nodes and pods, set up load balancers without the requirement of a command line tool or the editing of hard to read YAML files.
```

File: console/my-account/how-to/open-a-support-ticket.mdx (hunk 1 above)
Comment (jcirinosclwy) [inline console/my-account/how-to/open-a-support-ticket.mdx:16]:
```suggestion
Scaleway provides 24/7 basic technical support by ticket for free. You can contact our Support team directly from the console. You can [upgrade your support level](/console/my-account/how-to/configure-support-plans/) to benefit from phone support and a dedicated account manager.
```

File: tutorials/odoo-easy-deploy/index.mdx (hunk 7 above)
[the remark this answers is not in the document]
Comment (nerda-codes) [reply tutorials/odoo-easy-deploy/index.mdx:187]: I just checked again, I was convinced it wasn't capitalized but you're right my bad

File: dedibox-network/ipv6/how-to/enable-ipv6-slaac.mdx [hunk 8]
```diff
@@ -18,7 +18,7 @@ categories:
   - You have [created](/dedibox/dedicated-servers/how-to/order-a-server) a Dedibox dedicated server
 </Message>
 
-Activation of IPv6 SLAAC (stateless auto-configuration of addresses) assigns **one** `/128` IPv6 prefix to your server (one usable IPv6 address). This IP is statically linked to your server and can not be attributed to a another server.
+Activation of IPv6 SLAAC (stateless autoconfiguration of addresses) assigns **one** `/128` IPv6 prefix to your server (one usable IPv6 address). This IP is statically linked to your server and can not be attributed to another server.
 
 <Message tyoe="note">
```
Comment (jcirinosclwy) [inline dedibox-network/ipv6/how-to/enable-ipv6-slaac.mdx:23]:
```suggestion
<Message type="note">
```

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**